### PR TITLE
Added steps to install xray daemon on standalone EC2.

### PR DESCRIPTION
**Pull request by vnandwana**

### CODE REVIEW

**Review by reshmabidikar — APPROVED** [review]
I'm not familiar with this so cannot comment, approving the PR.

File: userguide/tutorials/metrics-cloudwatch.adoc [hunk 1]
```diff
@@ -60,8 +60,20 @@ AWS X-Ray is a service that collects and analyzes data from your applications. A
 To setup the daemon, log in to your EC2 instance and run the following commands:
 
 ```
-curl https://s3.dualstack.us-east-1.amazonaws.com/aws-xray-assets.us-east-1/xray-daemon/aws-xray-daemon-3.x.deb -o /home/ubuntu/xray.deb
-sudo dpkg -i /home/ubuntu/xray.deb
+# Download the X-Ray daemon package
```
Comment (sbrossie) [inline userguide/tutorials/metrics-cloudwatch.adoc]: Would it make sense to include this by default in our AMIs instead? @pierre @vnandwana Thoughts?

Comment (pierre) [reply userguide/tutorials/metrics-cloudwatch.adoc]: Sounds good! 👍

Comment (vnandwana) [reply userguide/tutorials/metrics-cloudwatch.adoc]: Yes, I agree. I'm going to start working on this.

Comment (vnandwana) [reply userguide/tutorials/metrics-cloudwatch.adoc]: XRay daemon now installs using Ansible script. PR -- https://github.com/killbill/aws-marketplace/pull/202

Comment (vnandwana) [reply userguide/tutorials/metrics-cloudwatch.adoc]: Closing this PR in favour of https://github.com/killbill/killbill-docs/pull/550

**Comment by vnandwana** [comment]
Closing this PR in favour of https://github.com/killbill/killbill-docs/pull/550

**Pull request closed without merging**